src/js/client/controls.js

```diff
@@ -0,0 +1,150 @@
+// Present a standard set of user controls with login/logout/profile links
+// TODO: Convert this to use a common "toggler" grade once that has been sufficiently exercised and moved somewhere central.
+/* global fluid, jQuery */
+(function ($) {
+    "use strict";
+    var gpii = fluid.registerNamespace("gpii");
+    fluid.registerNamespace("gpii.express.user.frontend.controls");
+
+    gpii.express.user.frontend.controls.handleMenuKeys = function (that, event) {
```

Comment:
I was able to update the component to use both `activatable` and `selectable` and removed most of my unique event handling bits in favor of those standard approaches.